styles/Events.module.scss

```diff
@@ -0,0 +1,19 @@
+.container {
+  @apply w-full;
+  @apply bg-color-3;
+  // header Styling
+  header {
+    @apply w-full;
+    @apply px-48 py-12;
+    @apply bg-color-1;
+    span {
+      @apply max-w-4xl;
+      @apply max-[600px]:text-xs;
+    }
+    // Mobile
+    @apply max-[900px]:px-32;
+    @apply max-[850px]:px-16;
+    @apply max-[700px]:px-10;
+    @apply max-[450px]:px-8;
```

Comment:
im really not a fan of this many breakpoints, its fine for now but lets revisit at some later point